Trainer_v5/Main.cs

```diff
@@ -33,11 +38,15 @@ public static void CloseSettingsWindow()
 
 		public static void CreateUIButtons()
 		{
+			_traitChangeWindow = new EmployeeTraitChangeWindow();
```

Comment:
Since the method is called CreateUIButtons, it should be better if it's only doing that. EmployeeTraitChangeWindow method should be called from somewhere else.